### ISSUE TITLE
Vendure 2.0.0 - Custom field UI regarding facet-value-form-input

### ISSUE THREAD

**Issue by BHO010** [vendure-ecommerce/vendure#2216]
**Describe the bug**
Adding a custom field facet form input UI to seller entity, values of the facet not being captured on creation. Values are of incorrect type when updating. 

**To Reproduce**
1. Go to vendure.config.ts and add a custom field on the seller entity.
2. In the admin panel, create a seller. You should be able to see the facet form input UI in the custom field and search for the facet and add them.
3. Click on the create button to create a seller. However, the custom field values are not captured but the seller is being created
4. Next go to the sellet list and edit the newly created seller. Try to add the custom field again and this time the error appear.
5. The custom field facets value seems to be an array of objects but it requires an array of strings. 

**Expected behavior**
Being able to create a seller with the custom field values being captured correctly.
Being able to update seller custom field correctly. 

**Environment (please complete the following information):**
 - @vendure/core version: v2.0.0
 - Nodejs version: v16.14.0
 - Database (mysql/postgres etc): mysql

**Additional context**
Image for step 1
![ss3](https://github.com/vendure-ecommerce/vendure/assets/54208795/0026186c-021b-42ee-9b54-33d3465d9b54)

Image for step 2/3
![ss1](https://github.com/vendure-ecommerce/vendure/assets/54208795/24d63aa1-0cce-4364-8de7-7cb99d056d9f)

Image for step 4/5
![ss2](https://github.com/vendure-ecommerce/vendure/assets/54208795/5ca53ae6-6862-4421-a321-08d3d8b375a8)

**Comment by michaelbromley** [comment]
Hi, thanks for the report!

The correct custom field config for setting up a relation to the FacetValue entity would look like this:

```ts
customFields: {
        Seller: [
            {
                name: 'category',
                type: 'relation',
                entity: FacetValue,
                list: true,
                ui: {
                    component: 'facet-value-form-input',
                },
            },
        ],
    },
```

However, in investigating this issue I did find some actual issues where custom field relation logic had not been implemented in the Seller service, so this will be fixed in the upcoming 2.0.1 patch.

**Comment by BHO010** [comment]
ah, I see. Should have tried using relation instead of text. Thanks for the help.

Looking forward to patch 2.0.1
